viz_scripts/generic_metrics.ipynb

```diff
@@ -56,21 +57,27 @@
     "%matplotlib inline"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "eaf90438",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Do not run this notebook if it does not have trip-labels as MULTILABEL; nbclient will run up through this cell\n",
```

Comment:
I'm fairly certain that not all multilabel deployments have survey_info, could we change this to if `"trip-labels": "ENKETO"` so that only the survey deployments get stopped?

Reply:
changed to 
```
# Do not run this notebook if it has trip-labels as ENKETO nbclient will run up through this cell
if survey_info.get('trip-labels', None) == 'ENKETO':
    ipython = get_ipython()
    ipython._showtraceback = scaffolding.no_traceback_handler
    raise Exception("The plots in this notebook are not relecant for ENKETO trip-labels")
```